apps/web/components/settings/platform/dashboard/managed-user-list/index.tsx

```diff
@@ -0,0 +1,84 @@
+import type { PlatformOAuthClient } from "@calcom/prisma/client";
+import { EmptyScreen } from "@calcom/ui";
+
+import type { ManagedUser } from "@lib/hooks/settings/platform/oauth-clients/useOAuthClients";
+
+import { OAuthClientsDropdown } from "@components/settings/platform/dashboard/oauth-client-dropdown";
+
+type ManagedUserListProps = {
+  oauthClients: PlatformOAuthClient[];
+  managedUsers?: ManagedUser[];
+  initialClientName: string;
+  initialClientId: string;
+  isManagedUserLoading: boolean;
+  handleChange: (clientId: string, clientName: string) => void;
+};
+
+export const ManagedUserList = ({
+  initialClientName,
+  initialClientId,
+  oauthClients,
+  managedUsers,
+  isManagedUserLoading,
+  handleChange,
+}: ManagedUserListProps) => {
+  return (
+    <div>
+      <div className="border-subtle mx-auto block justify-between rounded-t-lg border px-4 py-6 sm:flex sm:px-6">
+        <div className="flex w-full flex-col">
+          <h1 className="font-cal text-emphasis mb-1 text-xl font-semibold leading-5 tracking-wide">
+            Managed Users
+          </h1>
+          <p className="text-default text-sm ltr:mr-4 rtl:ml-4">
+            See all the managed users created by your OAuth client.
+          </p>
+        </div>
+        {Array.isArray(oauthClients) && oauthClients.length > 0 && (
+          <OAuthClientsDropdown
+            oauthClients={oauthClients}
+            initialClientName={initialClientName}
+            handleChange={handleChange}
+          />
+        )}
+      </div>
+      {Array.isArray(managedUsers) && !isManagedUserLoading && managedUsers.length ? (
```

Comment:
Code from line 44 to 82 could be a separate component e.g. `<Users>` and then code above from line 26 to 43 `<Header>` or something - this is just a reminder that its easier to read code when its broken down into smaller parts / components.